foundation/aissemble-foundation-core-python/pyproject.toml

```diff
@@ -35,6 +35,9 @@ nose = ">=1.3.7"
 kappa-maki = ">=1.0.1"
 pylint = "^3.1.0"
 
```

Comment:
I (Informational): added this to basically force just about every to stick to 2.31.0